### DISCUSSION TITLE
What if I want to train 7 speakers at the same time, but the training set for each speaker is different?

### DISCUSSION THREAD

**Discussion by pkuhsczy** [coqui-ai/TTS#3045] [General Q&A]
the trainging set is : 939, 4500, 303, 296, 333, 2012, 462 seperately, do I need to wirte something to balance them ?

**Comment by pkuhsczy** [comment]
it is about fine-tuning using models_en_vctk_vits

**Comment by erogol** [comment]
You can merge datasets and TTS can help you with that. Check out the recipes.

**Reply by pkuhsczy** [reply]
Thank you very much. After the merge , do I need to set weighted_sampler_multipliers={" speaker_id1 ":, "speaker_id2"... }?
